CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,13 @@
+# Contributor Code of Conduct
+
+As contributors and maintainers of this Stencil Community project, we pledge to respect everyone who contributes by posting issues, updating documentation, submitting pull requests, providing feedback in comments, and any other activities.
+
+Communication through this repository must be constructive and never resort to personal attacks, trolling, public or private harassment, insults, or other unprofessional conduct.
+
+We promise to extend courtesy and respect to everyone involved in this project regardless of gender, gender identity, sexual orientation, disability, age, race, ethnicity, religion, or level of experience. We expect anyone contributing to Stencil Community projects to do the same.
+
+If any member of the community violates this code of conduct, the maintainer(s) and/or the Ionic team may take action, including but not limited to removing issues, comments, and PRs or blocking accounts as deemed appropriate.
+
+If you are subject to or witness unacceptable behavior, or have any other concerns, please contact the maintainer(s) or email contact@ionic.io.
+
+To review the full Ionic community guidelines, please visit the following url: https://github.com/ionic-team/ionic-framework/blob/main/CODE_OF_CONDUCT.md
```

Comment:
Should this point to the Stencil CoC? https://github.com/ionic-team/stencil/blob/main/CODE_OF_CONDUCT.md?

IDK if that also necessitates a change the phrase 'Ionic community guidelines' (to 'Stencil community guidelines')

Reply:
Gotcha. I think aligning the two CoC's makes sense, since both projects (Stencil & Framework) exist as a part of Ionic's OSS offerings.